### PR TITLE
Replace PaperClip for ActiveStorage and Add ENV variable

### PR DESCRIPTION
**Pull request by fernandoamz**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -5,9 +5,21 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable,
          :omniauthable, omniauth_providers: [:facebook]
         
-  has_attached_file :avatar, styles: { medium: '300x300>', thumb: '100x100>' }
+  has_one_attached :avatar
```
Comment (8geonirt) [inline app/models/user.rb]: Remove this trailing space (the one at the end of the line), sometimes with just a single space like this Git will detect it as a change.

File: app/models/user.rb [hunk 2]
```diff
@@ -5,9 +5,21 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable,
          :omniauthable, omniauth_providers: [:facebook]
         
-  has_attached_file :avatar, styles: { medium: '300x300>', thumb: '100x100>' }
+  has_one_attached :avatar 
   has_many :todo_lists
-  validates_attachment_content_type :avatar, content_type: /\Aimage\/.*\Z/
+  validate :avatar_validation
+
+  def avatar_validation
```
Comment (8geonirt) [inline app/models/user.rb:12]: Well done, we need to wait until ActiveStorage gets validations out of the box.

Or... try to create a contribution for that 🤓 😅

**Review by 8geonirt — CHANGES_REQUESTED** [review]
Nice job, just address the only one comment I left

**Review by 8geonirt — APPROVED** [review]
🤓 👍

**Review by diegosaul402 — APPROVED** [review]
LGTM~ :shipit:

**Comment by pacocd** [comment]
oh man, I was just starting my review ☹️

**Comment by fernandoamz** [comment]
Sorry i can reopen the branch ?  because it's merged  :c

**Comment by pacocd** [comment]
lol, no worries! haha

**Pull request merged**